KsApi/models/graphql/adapters/Comment+GraphComment.swift

```diff
@@ -4,15 +4,25 @@ extension Comment {
   /**
    Returns a minimal `Comment` from a `GraphComment`
    */
+
+  /**
+   FIXME: Some of the properties needed for `Comment` hasn't been mapped out on GraphComment. We are returning default values.
+   These properties are `Author.imageUrl`, `authorBadges`, `createdAt`, `deletedAt`, `isDeleted`
+   */
   static func comment(from graphComment: GraphComment) -> Comment {
     return Comment(
       author: Author(
         id: graphComment.author.id,
+        imageUrl: "",
         isCreator: graphComment.author.isCreator,
         name: graphComment.author.name
       ),
+      authorBadges: nil,
       body: graphComment.body,
+      createdAt: TimeInterval.init(),
```

Comment:
I'm a little unclear about the `FIXME` comments I'm seeing related to the mapping of Comment to GraphComment. Is there a reason we are defaulting values instead of making them optionals set to nil?